README.md

```diff
@@ -169,3 +169,13 @@ jormungandr start --genesis-config genesis.yaml \
   --secret key.xprv
 ```
 
+# License
+
+This project is licensed under either of the following licenses:
+
+ * Apache License, Version 2.0, ([LICENSE-APACHE](LICENSE-APACHE) or
+   http://www.apache.org/licenses/LICENSE-2.0)
+ * MIT license ([LICENSE-MIT](LICENSE-MIT) or
+   http://opensource.org/licenses/MIT)
+
+Please choose the licence you want to use.
```

Comment:
I think it could use a more permissive language here, like "at your option", like advised [in The Guidelines](https://rust-lang-nursery.github.io/api-guidelines/necessities.html#crate-and-its-dependencies-have-a-permissive-license-c-permissive).